src/components/Layout/Layout.js

```diff
@@ -48,7 +48,7 @@ class ScrollDetector extends React.Component {
   }
 
   render() {
-    const { className, element, ...rest } = this.props
+    const { className, element, offsetHeight, ...rest } = this.props
```

Comment:
@iamvictorli @tkarssli @zach-ethos guys, I had to do this to prevent:

![image](https://user-images.githubusercontent.com/142403/68050250-befada80-fcb2-11e9-8261-ddca4282414a.png)

Which seems to reflect [transferring props](https://reactjs.org/warnings/unknown-prop.html) ... I believe this is fine since it seems we only need to use this in the component itself for:

```js

  updateScrollState = () => {
    const scrollTop = window.pageYOffset || window.document.scrollTop
    this.setState({ isScrolled: scrollTop > this.props.offsetHeight })
  }
```

Just a heads up though!

Reply:
This is an issue that we see on CMS. Thanks for looking at this